vm/actor/src/builtin/verifreg/types.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use crate::StoragePower;
+use address::Address;
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct VerifierParams {
+    pub address: Address,
+    pub allowance: Datacap,
+}
+
+pub type AddVerifierParams = VerifierParams;
+pub type AddVerifierClientParams = VerifierParams;
+
+pub const MINIMUM_VERIFIED_SIZE: u32 = 1 << 20;
```

Comment:
nit: can you move this to the top (nice to have constants always defined at top of a file)